readme.md

```diff
@@ -10,6 +10,21 @@ Package maintainers will be able to publish JavaScript files to NPM in this form
 
 Do not confuse this format with an AST. It is designed purely for interoperability and publishing. Using this format you can build an AST or regular CSS or any other structure.
 
+### Glossary
+
+The following are some terms that are used throughout the spec, or are relevant when discussing the spec.
+
+- **Markers:** Like nodes in an AST format, these tuples indicate significant literals and values (See [Markers](#markers))
+- **References:** References to primitives in the host language, typically known as "interpolations" in tagged template literals in JavaScript
+- **ISTF array:** An array of markers representing the CSS input, including references
```

Comment:
btw. thinking about this one:
1. is ISTF array the same like entire ISTF?
2. If it is, it can contain also selectors, right?

Do we need a term for istf array which contains only the declarations?

Reply:
mb: ISTF array - entire ISTF set
ISTF declarations - only part of ISTF which allows everything except of selectors, basically props/values

Reply:
I think it is used already in an unclear way

Reply:
yea, I think that's unfortunately ambiguous: ISTF declarations could both be the marker types, or the part of the array containing declarations :/

Reply:
I got it: istf rule and istf declaration, same like css rule and css declaration.
CSS rule is selector + declaration. Same here.

Reply:
so lets add 3:

ISTF, ISTF rule, ISTF declaration. where ISTF contains ISTF rules, and ISTF rule contains ISTF selectors and ISTF declarations.